iceoryx_hoofs/test/moduletests/test_cxx_stack.cpp

```diff
@@ -250,8 +250,8 @@ TEST_F(stack_test, CopyConstructorWorksAndCallsTestClassCopyConstructor)
 
     cxx::stack<TestClass, STACK_SIZE> testStack(m_sut);
     EXPECT_THAT(TestClass::copyCTor, Eq(1));
-    EXPECT_THAT(testStack.size(), Eq(1));
-    EXPECT_THAT(testStack.pop(), Eq(TestClass(ELEMENT, ELEMENT, ELEMENT)));
+    ASSERT_THAT(testStack.size(), Eq(1));
```

Comment:
Note that here I am willing to risk a test crash for the sake of brevity (avoid a result variable). The size might be wrong or there could be a bug, but the same could be said for an additional `hasValue` check. 

If there is a serious bug a crash can never be avoided (nor can a silent failure ...).

Nothing to do here.

Reply:
Noted.